docs/accordproject-developers.md

```diff
@@ -3,4 +3,22 @@ id: accordproject-developers
 title: For Developers
 ---
 
-**TBD**
+## Why is the Accord Project relevant for developers?
+The Accord Project provides a universal format for smart legal contracts, and this format is embodied in a variety of open source projects that comprise the Accord Project technology stack. Input from developers are crucial for the Accord Project.
+
+Developers can contribute by translating legal text into corresponding computer code, creating Accord Project templates to be used by lawyers and businesses. In addition, developers can provide input on the development of the technology stack components: language, models, templating, and other tools.
+
+If this interests you, please visit our [Technology Working Group](https://www.accordproject.org/working-groups/technology) page, and join our [slack channel](https://accord-project-slack-signup.herokuapp.com/)!
+
+
+## How to use this documentation?
+
+If you are new to the Accord Project, please read the [Key Concepts](accordproject-concepts) page. This will allow you to understand the three components of a template (text, model, and logic) and how they work together.
+
+If you want to create a template yourself, please see the [Legal-Tech Tutorials - Authoring in Template Studio](tutorial-latedelivery) page for a step-by-step guide on how to create your first template on template studio. If you want to author, validate, and run Accord Project templates _locally_, please visit our [Install Cicero](https://docs.accordproject.org/docs/next/started-installation.html) page for instructions.
```

Comment:
I would split this part three paragraphs:
1. a paragraph pointing to the getting started guide (same link) to quickly get up and running as a developer
2. the additional information for legal tech and 
3. the `dive into our technology stack` part as is